core/swift54Action/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 #
 -->
 
-# Apache OpenWhisk Swift 5.4 Runtime Container
+# Apache OpenWhisk Swift 5.7 Runtime Container
```

Comment:
Minor nit; this file shouldn't change.
```suggestion
# Apache OpenWhisk Swift 5.4 Runtime Container
```

Reply:
https://github.com/apache/openwhisk-runtime-swift/commit/4917c54a630c82b33a68ec0d66139b1657652ebe